runtime/bcverify/vrfyhelp.c

```diff
@@ -906,7 +906,10 @@ isFieldAccessCompatible(J9BytecodeVerificationData *verifyData, J9ROMFieldRef *f
 		J9BranchTargetStack *liveStack = (J9BranchTargetStack *)verifyData->liveStack;
 		J9ROMFieldShape *field = findFieldFromCurrentRomClass(romClass, fieldRef);
 #if defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES)
-		IDATA isStrictField = (NULL != field) && J9_ARE_ALL_BITS_SET(field->modifiers, J9AccStrict);
+		IDATA isStrictField =
+			(NULL != field)
+			&& J9_IS_CLASSFILE_OR_ROMCLASS_VALUETYPE_VERSION(romClass)
```

Comment:
I suggest to add a macro `J9ROMFIELD_IS_CONTENDED` and include the class version check in the macro.

Reply:
Where is the language `IS_CONTENDED` used for strict fields?

Reply:
Sorry about the typo. `J9ROMFIELD_IS_CONTENDED` -> `J9ROMFIELD_IS_STRICT`